docs/source/features/packaging_output_models.md

```diff
@@ -46,6 +46,30 @@ The inference config file is a json file including `execution_provider` and `ses
 #### SampleCode
 Olive will only provide sample codes for ONNX model. Sample code supports 3 different programming languages: `C++`, `C#` and `Python`. And a code snippet introducing how to use Olive output artifacts to inference candidate model with recommended inference configurations.
 
+### AzureMLModels
+AzureMLModels packaging will register the output models to your Azure Machine Learning workspace. The asset name will be set as `<packaging_config_name>_<accelerator_spec>_<model_rank>`. For instance, if the output model is ONNX model and the packaging config is:
+
+```
+{
+    "type": "AzureMLModels",
```

Comment:
`AzureMLModels` or `AzureMLModel`? Maybe the singular is better?

Reply:
I'm using the same name as how AML calls this. Actually both `AzuerMLModels` and `AzureMLData` here indicate the AML assets tag name.
![image](https://github.com/microsoft/Olive/assets/85524621/6b2d60be-983a-4958-a1d3-08a25474137d)